app/components/channel_icon/index.tsx

```diff
@@ -196,4 +196,4 @@ const ChannelIcon = ({
     );
 };
 
-export default ChannelIcon;
+export default React.memo(ChannelIcon);
```

Comment:
There seem to be some performance issues when rendering the channel icon, so adding the memo solved it.